plugins/imu/IMU.cc

```diff
@@ -25,19 +25,19 @@ using namespace systems;
 namespace gzyarp
 {
 
-class IMU : public System,
+class Imu : public System,
```

Comment:
Not blocking, but just fyi `IMU` is a acronym (Inertial Measurement Unit), so that's the reason of why it was all capital. Anyhow, I seem that upstream `gz-sim` uses `Imu` style, so it is ok to name it like that.

Reply:
Yep that was one of the reasons for which I updated the class name, if you prefer I can revert it to the previous version :)

Reply:
No no that is ok, I just wanted to understand what was happening, but I realized it while writing my comment.